doc/rbd/rbd-mirroring.rst

```diff
@@ -83,23 +83,72 @@ explicitly.
 
 For example::
 
-        $ rbd --cluster local mirror pool disable image-pool
-        $ rbd --cluster remote mirror pool disable image-pool
+        $ rbd --cluster site-a mirror pool disable image-pool
+        $ rbd --cluster site-b mirror pool disable image-pool
 
-Add Cluster Peer
-----------------
+Bootstrap Peers
+---------------
 
 In order for the ``rbd-mirror`` daemon to discover its peer cluster, the peer
-needs to be registered to the pool. To add a mirroring peer Ceph cluster with
-``rbd``, specify the ``mirror pool peer add`` command, the pool name, and a
-cluster specification::
+needs to be registered to the pool and a user account needs to be created.
+This process can be automated with ``rbd`` and the
+``mirror pool peer bootstrap create`` and ``mirror peer peer bootstrap import``
+commands.
+
+To manually create a new bootstrap token with ``rbd``, specify the
+``mirror pool peer bootstrap create`` command along with an optional friendly
+site name to describe the local cluster::
+
+        rbd mirror pool peer bootstrap create [--site-name {local-site-name}]
```

Comment:
nit: I am not very happy the command name uses `pool`. A command starting with `mirror pool` suggests (from previous experience) that it is applied for a pool only. In this case as the pool is not specified one could guess that it is applied for the default pool. But actually it is applied for the whole cluster.

So, `rbd mirror peer bootstrap create` or something like this could be nice for me. But if you like your current version just ignore this.

Reply:
I don't like it either to be honest. I had originally just wanted "mirror pool bootstrap" (i.e. bootstrap as a verb instead of a noun) and "mirror pool peer import", but Sage like the idea of mirroring the commands to pair them together. Perhaps "mirror bootstrap create" is a close-enough compromise.

Reply:
... or now that I am starting to change the APIs, perhaps keep "mirror pool peer bootstrap create", force it to take a pool (IoCtx), and then use that to verify that mirroring it actually enabled on the pool prior to creating a bootstrap token. It already only creates a single user account anyway.

Reply:
"mirror bootstrap create"  would be good to me, but probably your last suggestion is even better :-)